src/index.css

```diff
@@ -1,26 +1,162 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 :root {
   --title: #0290ff;
   --link: #121212;
+  --panel: #fafafa;
+}
+
+.single-book,
+.category {
+  background: var(--panel);
+  margin: 20px 5%;
+  box-shadow: 8px 8px 25px 0 rgba(0, 0, 0, 0.5);
+}
+
+.book-category {
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-title {
+  margin: 0.188rem 20.188rem 0 0;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.book-author {
+  margin: 0.25rem 0 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
 }
 
-.header {
+.book-data {
   display: flex;
-  justify-content: center;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px;
+  list-style-type: none;
+}
+
+header {
+  display: flex;
+  justify-content: flex-start;
   align-items: center;
+  gap: 20px;
   color: var(--title);
+  margin: 20px 5%;
 }
```

Comment:
- [ ] Kindly use the logo on the right side of the header to make it similar to the Figma design
### Logo
![logo](https://user-images.githubusercontent.com/81727988/175799581-7df3d6eb-ad5a-4378-9bee-a3128b32c574.png)